### ISSUE TITLE
Getting web JSCAD example to work locally 

### ISSUE THREAD

**Issue by sdaau** [jscad/OpenJSCAD.org#632]
## Expected Behavior

```
git clone https://github.com/jscad/OpenJSCAD.org OpenJSCAD.org_git
cd OpenJSCAD.org_git
# since it is expected openjscad will use XHR to fetch files, `firefox packages/web/index.html` would likely fail;
# so let's try a local server:
python3 -m http.server 8000
```
Visit page: http://localhost:8000/packages/web/index.html - expecting everything works in browser

## Actual Behavior

The code snippet part of the page says:

```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
        "http://www.w3.org/TR/html4/strict.dtd">
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
        <title>Error response</title>
    </head>
    <body>
        <h1>Error response</h1>
        <p>Error code: 404</p>
        <p>Message: File not found.</p>
        <p>Error code explanation: HTTPStatus.NOT_FOUND - Nothing matches the given URI.</p>
    </body>
</html>
```

... and python server logs:

```
...
127.0.0.1 - - [30/Jul/2020 08:25:01] code 404, message File not found
127.0.0.1 - - [30/Jul/2020 08:25:01] "GET /packages/web/examples/logo.jscad HTTP/1.1" 404 -
...
```

## Steps to Reproduce the Problem

(written above)

## Specifications

  - Version: commit 231d3c6
  - Platform: Ubuntu 18.04
  - Environment: (browser, local server, node.js etc): browser, Firefox 78.0.2  

## Quick fix:

The 404'd file is really not present on the system:

```
OpenJSCAD.org_git$ ls -la packages/web/examples/
ls: cannot access 'packages/web/examples/': No such file or directory
```

... and neither is the examples directory:

```
OpenJSCAD.org_git$ ls -la packages/web/examples
ls: cannot access 'packages/web/examples': No such file or directory
```

... however if we look up the file:

```
OpenJSCAD.org_git$ find . -name 'logo.jscad'
./packages/examples/logo.jscad
```
... turns out it is there, but not in `packages/web/examples` directory; it is instead in `packages/examples`.

So, we can just throw in a relative symlink to the `packages/examples` directory in `packages/web`:

```
OpenJSCAD.org_git$ ln -s ../examples packages/web/
```
 
... and now when loading http://localhost:8000/packages/web/index.html in browser, everything works.

**Comment by z3dev** [comment]
@sdaau please trade the README and CONTRIBUTE as there are links between the related modules, which are created by running some NPM commands.

**Comment by sdaau** [comment]
Many thanks @z3dev :

> README and CONTRIBUTE

I did look at the README briefly, but - even if I've programmed vanilla JS some years ago - I'm not quite into the whole node/npm ecosystem: tried it a couple of times, and found the package management a bit confusing, which is why I try to kinda avoid it ... so when I read this in the README:

>> `- web: self hosted: can be found [here](./packages/web)`

... I really got happy, thinking "well, I can try this without doing anything with npm", and so I kept on insisting to get it working without npm; and I'm quite happy it worked.

However, I did not see `packages/web/README.md`, and I can see there is a section there "`### Use within a Web Site (pre built files)`", which basically describes what I've done above.

So, I guess all is fine - everything is documented, it was just a bit difficult for me to find the right piece of info - so I'm closing this.

Thanks!
